libs/core/tests/unit_tests/test_tools.py

```diff
@@ -2186,7 +2186,7 @@ class Foo(BaseModelV2):
         )
 
     @tool(args_schema=Foo)
-    def foo(x):  # type: ignore[no-untyped-def]
+    def foo(x: list[int]) -> list[int]:
```

Comment:
Probably reasonable to not update this type signature -- this test is attempting to verify that all the information is getting picked up from the args_schema?

Reply:
Indeed!
Done in https://github.com/langchain-ai/langchain/pull/29271/commits/454b292ba46aff42779406d7c002589481f96f03